src/main/kotlin/com/cjbooms/fabrikt/generators/PropertyUtils.kt

```diff
@@ -86,7 +87,7 @@ object PropertyUtils {
                 ClassSettings.PolymorphyType.SUB -> {
                     if (this is PropertyInfo.Field && isPolymorphicDiscriminator) {
                         property.addModifiers(KModifier.OVERRIDE)
-                        val discriminators = maybeDiscriminator.getDiscriminatorMappings(modelName)
+                        val discriminators = maybeDiscriminator.getDiscriminatorMappings(schemaName, modelName)
```

Comment:
Probably, I messed this up and `schemaName` was all we needed. I'll have a look some time next week and see if I can remove the `modelName` which seems to be redundant here